rllib/env/multi_agent_episode.py

```diff
@@ -1704,50 +1704,84 @@ def get_state(self) -> Dict[str, Any]:
         return list(
             {
                 "id_": self.id_,
-                "agent_ids": self.agent_ids,
+                "agent_to_module_mapping_fn": self.agent_to_module_mapping_fn,
```

Comment:
Actually, let's make the state a dict. States should always be `Dict[str, Any]`. I think this is a leftover from the early DreamerV3 days :)